services/gateway/apis/http-version3/swagger/parameters/reward.json

```diff
@@ -0,0 +1,9 @@
+{
+    "blockHeight": {
+        "name": "height",
+        "in": "query",
+        "description": "Height of the block",
+        "type": "number",
+        "minimum": 0
+    }
+}
```

Comment:
Do we need this? Don't we already have a param definition for `height`?

Else:
```
{
    "blockHeight": {
        "name": "height",
        "in": "query",
        "description": "Block height to query.",
        "type": "number",
        "minimum": 0
    }
}
```

Reply:
That parameter accepts range as well. Updating description.
> Query by height or a height range (fromHeight:toHeight, both inclusive)